java/templates/s2i/run

```diff
@@ -3,21 +3,8 @@
 # Global S2I variable setup
 source `dirname "$0"`/s2i-setup
 
-# Setup Jolokia to accept basic auth, using a randomly generated password that is stored
```

Comment:
If we are going the route to update jolokia-opts in agent-bond instead of checking it in here (see previous comment), then the generation of the password should remain here since it is specific to the s2i setup (and not really connected to jolokia).

Or one could add it as an option to jolokia-opts like by setting a `AB_JOLOKIA_RANDOM_PASSWORD`